src/hotspot/cpu/aarch64/stubGenerator_aarch64.cpp

```diff
@@ -1975,7 +1975,7 @@ class StubGenerator: public StubCodeGenerator {
     __ br(Assembler::EQ, L_done_pop);
 
     __ BIND(L_do_card_marks);
-    bs->arraycopy_epilogue(_masm, decorators, is_oop, start_to, count_save, rscratch1, wb_post_saved_regs);
+    bs->arraycopy_epilogue(_masm, decorators, is_oop, noreg, start_to, count_save, rscratch1, wb_post_saved_regs);
```

Comment:
You should allocate another register to store `from`, and pass it to `arraycopy_epilogue`.

Please see
https://github.com/mmtk/jdk/blob/d60b78e0c199943c20b852f0036c4533285a308c/src/hotspot/cpu/riscv/stubGenerator_riscv.cpp#L1512
https://github.com/mmtk/jdk/blob/d60b78e0c199943c20b852f0036c4533285a308c/src/hotspot/cpu/riscv/stubGenerator_riscv.cpp#L1519
https://github.com/mmtk/jdk/blob/d60b78e0c199943c20b852f0036c4533285a308c/src/hotspot/cpu/riscv/stubGenerator_riscv.cpp#L1577
https://github.com/mmtk/jdk/blob/d60b78e0c199943c20b852f0036c4533285a308c/src/hotspot/cpu/riscv/stubGenerator_riscv.cpp#L1621